objects-api/src/main/kotlin/com/ritense/objectsapi/service/ObjectsApiService.kt

```diff
@@ -71,25 +88,32 @@ open class ObjectsApiService(
      * @param type the <code>type name as String</code> to filter
      */
     fun getObjects(type: URI?, searchParams: List<ObjectSearchParameter> = emptyList()): Collection<Object> {
-        return RequestBuilder
-            .builder()
-            .baseUrl(objectsApiProperties.objectsApi.url)
-            .token(objectsApiProperties.objectsApi.token)
-            .path("${ObjectsApiConnector.rootUrlApiVersion}/objects")
-            .get()
-            .queryParams(searchParams.associate { "data_attrs" to it.toQueryParameter() }.toMutableMap())
-            .queryParam("type", type)
-            .executeForCollection(Object::class.java)
+        var next = true
+
+        return generateSequence(1) { i -> if (next) i + 1 else null }
```

Comment:
Why this structure? Feels kind of difficult to read.

Reply:
Please check again. I changed the `getZaakObjecten` as well.